cpu/em_smtfolding.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
```

Comment:
Remove empty line and add at https://github.com/avocado-framework-tests/avocado-misc-tests/pull/808/files#diff-46a112795b6b8d0b2433d6eec30327a9R15